Library/Formula/norm.rb

```diff
@@ -1,8 +1,9 @@
 class Norm < Formula
   desc "NACK-Oriented Reliable Multicast"
   homepage "https://www.nrl.navy.mil/itd/ncs/products/norm"
-  url "https://downloads.pf.itd.nrl.navy.mil/norm/archive/src-norm-1.5b3.tgz"
-  sha256 "ec2014f57be12ea3ade0685faa1e6e161f168311c8975093bce33a6e1dc5e77c"
+  url "https://downloads.pf.itd.nrl.navy.mil/norm/archive/src-norm-1.5r5.tgz"
+  version "1.5r5"
```

Comment:
FYI - I have now posted src-norm-1.5r6.tgz that has an important bug fix.

For future reference, is there a certain way I should report such key updates to homebrew?

best regards,

Brian


> On Jun 29, 2015, at 6:13 PM, Hans Duedal <notifications@github.com> wrote:
> 
> In Library/Formula/norm.rb <https://github.com/Homebrew/homebrew/pull/41181#discussion_r33520255>:
> 
> > @@ -1,8 +1,9 @@
> >  class Norm < Formula
> >    desc "NACK-Oriented Reliable Multicast"
> >    homepage "https://www.nrl.navy.mil/itd/ncs/products/norm"
> > -  url "https://downloads.pf.itd.nrl.navy.mil/norm/archive/src-norm-1.5b3.tgz"
> > -  sha256 "ec2014f57be12ea3ade0685faa1e6e161f168311c8975093bce33a6e1dc5e77c"
> > +  url "https://downloads.pf.itd.nrl.navy.mil/norm/archive/src-norm-1.5r5.tgz"
> > +  version "1.5r5"
> Unfortunately it detects "5" as the version.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/Homebrew/homebrew/pull/41181/files#r33520255>.
>